puppet/zulip_ops/files/nginx/sites-available/loadbalancer

```diff
@@ -19,6 +19,23 @@ server {
     include /etc/nginx/zulip-include/certbot;
 }
 
+server {
+    # The listen needs to be `www.zulipstaging.com` since bare zulipstaging.com
+    # is not a CNAME and thus has the public IP inside EC2
```

Comment:
This comment seems not as applicable as in the below block, given that the `listen` matches the `server_name` here.

Reply:
Indeed, deleted.